src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -498,7 +498,14 @@ public Toolbar getToolbar()
     {
         if (toolbar == null)
         {
-            toolbar = new Toolbar(this);
+            boolean usingNewToolbar = (boolean) executeScript
+                ("return interfaceConfig._USE_NEW_TOOLBOX");
```

Comment:
i don't think we ever break lines like this in java.  at least put the opening brace of the method call next to the `executeScript`:
```
boolean usingNewToolbar = (boolean)executeScript(
    "return interfaceConfig._USE_NEW_TOOLBOX");
```
but i think even better (if it fits) would be:
```
boolean usingNewToolbar = 
    (boolean)executeScript("return interfaceConfig._USE_NEW_TOOLBOX");
```

Reply:
I went with the first, line breaking after executeScript, due the second exceeding the 80 character line limit.

Reply:
hmm I think we used to put the = sign on the next line (at least I do that not sure if it's part of the official style guidelines)